spec/controllers/users_controller_spec.rb

```diff
@@ -45,7 +49,29 @@
       end
     end
 
-    context "with invalid input" do
+    context "with valid personal info and declined card" do
+      let(:charge) { double(:charge, successful?: false, error_message: 'Your card was declined.') }
```

Comment:
You are using a charge object, which will stub error_message - as you can see it comes from the StripeWrapper, you won't rescue exception in controller but in Wrapper _ i commented on this above already but here you do what you are supposed to.